README.md

```diff
@@ -170,13 +170,22 @@ HEAP_DUMP_PATH 	| Specify a location for a heap dump using `XX:HeapDumpPath` | `
+--- 				|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| ---
+CASSANDRA_CLUSTER	| Enable connection to an external Cassandra cluster                                                                                                                                                                                       | `false`
+CASSANDRA_NODES		| A comma separated list of C* nodes (e.g. `10.20.205.26,10.20.205.233`)                                                                                                                                                                   |
+CASSANDRA_PORT		| C* port		                                                                                                                                                                                                                                | `9042`
+CASSANDRA_USERNAME	| C* username	                                                                                                                                                                                                                             |
+CASSANDRA_PASSWORD	| C* password	                                                                                                                                                                                                                             |
+CASSANDRA_ASYNC_PROCESSING_ENABLED | Enable asynchronous processing of records in DDS Dataset save operation. Failures to store individual records will not interrupt operation execution.                                                                                    | `false`
+CASSANDRA_KEYSPACES_PREFIX | Prefix to use when creating Pega-managed keyspaces in Cassandra                                                                                                                                                                          |
+CASSANDRA_EXTENDED_TOKEN_AWARE_POLICY | When a Cassandra range query runs, the extended token aware policy selects a token from the token range to determine the Cassandra node to which to send the request, which is effective when the token range partitioner is configured. | `false`
+CASSANDRA_LATENCY_AWARE_POLICY | When used, this policy will collect the latencies of the queries to each Cassandra node and maintain a per-node latency score (an average).                                                                                              | `false`
+CASSANDRA_CUSTOM_RETRY_POLICY | Decides if to use custom pega retry policy. Number of retires can be customized using DSS value: `dnode/cassandra_custom_retry_policy/retryCount`                                                                                                                                        | `false`
```

Comment:
I think we can tone down this one a bit. Since someone could deploy their pega cluster first with having this enabled (and that's probably the preferred way for helm charts) it is fine for them to configure it after. What about this?

Enable the use of a customized retry policy for your Pega Platform deployment. After enabling this policy number of retires in Pega Platform should be configured using the dynamic system setting (DSS): `dnode/cassandra_custom_retry_policy/retryCount`, default value is `1`. When enabled, your deployment will retry using the custom value you set in the DSS.